scripts/dockerbot/pytestbase/setup.sh

```diff
@@ -62,13 +62,25 @@ do_or_die () {
     done
 }
 
+
+gate() {
+	log "gating $2 to $ARTIFACTOR_DIR/$1"
+	eval "$2" >> $ARTIFACTOR_DIR/$1 2>&1 
+	local RES=$?
+	if [ "$RES" -ne "0" ]
+	then
+		log "failed"
+		exit $RES
+	fi
+}
+
 # Runs pip update - optionally can make use of wheelhouse
 run_pip_update () {
+	export PYCURL_SSL_LIBRARY=nss
     if [ -n "$WHEEL_HOST_URL" ]; then
-        run_n_log "PYCURL_SSL_LIBRARY=nss pip install --trusted-host $WHEEL_HOST -f $WHEEL_HOST_URL -Ur $CFME_REPO_DIR/requirements/frozen.txt --no-cache-dir"
-    else
-        run_n_log "PYCURL_SSL_LIBRARY=nss pip install -Ur $CFME_REPO_DIR/requirements/frozen.txt --no-cache-dir"
+        export PIP_TRUSTED_HOST="$WHEEL_HOST" PIP_FIND_LINKS="$WHEEL_HOST_URL"
```

Comment:
**REQUIRED:** Does this mean we must have a wheel host now?

Reply:
no